src/dotnetaspire/install.sh

```diff
@@ -13,22 +13,22 @@ set -e
 # default to latest if not specified
 VERSION="${VERSION:-"latest"}"
 
-if [[ ! $VERSION =~ ^(9\.0|latest|latest-daily)$ ]]; then
-    echo "Error: VERSION must be either '9.0', '9.0.0', 'latest', or 'latest-daily' not: '$VERSION'."
+if [[ ! $VERSION =~ ^(9\.1|latest|latest-daily)$ ]]; then
+    echo "Error: VERSION must be either '9.1', '9.1.0', 'latest', or 'latest-daily' not: '$VERSION'."
     exit 1
 fi
 
-if [[ $VERSION =~ ^(9\.0|9\.0\.0|latest)$ ]]; then
-    VERSION="9.0.0"
+if [[ $VERSION =~ ^(9\.1|9\.1\.0|latest)$ ]]; then
+    VERSION="9.1.0"
 fi
 
 echo "Activating feature '.NET Aspire' version: $VERSION"
 
-# Before .NET Aspire 9.0 install required `dotnet workload`: this is no longer necessary, as Aspire is 
+# Before .NET Aspire 9.1 install required `dotnet workload`: this is no longer necessary, as Aspire is 
 # installed when restoring Aspire projects. It's only necessary to install the appropriate version of the templates.
 
 
-if [[ $VERSION =~ ^(9\.0\.0)$ ]]; then
+if [[ $VERSION =~ ^(9\.1\.0)$ ]]; then
```

Comment:
NIT: I know you didn't add this in this PR, but might be useful to also pass in the `--force` on line 32 as well. That is required in case the workload is installed in order to correctly override the templates.

Reply:
Actually, that said, this will only run on brand new images which will of course won't have the workload, so feel free to disregard, and in fact, you could drop the `--force` from line 46 for the same reason.